aten/src/ATen/native/quantized/cpu/qthreshold.cpp

```diff
@@ -0,0 +1,45 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <torch/library.h>
+#include <ATen/quantized/Quantizer.h>
+#include <ATen/native/quantized/cpu/quantized_ops.h>
+
+#include <algorithm>
+
+namespace at {
+namespace native {
+
+DEFINE_DISPATCH(qthreshold_stub);
+
+namespace {
+  // the underlying implementation for quantized threshold kernel
+  Tensor quantized_threshold_impl(
+      const Tensor& qx,
+      Scalar threshold,
+      Scalar value
+  ) {
```

Comment:
nit: Be consistent

```
      Scalar value) {
```

Reply:
I've added the change to the latest commit!